src/com/pentalog/pentastagiu/homework2_bookCandyShop/MainClass.java

```diff
@@ -0,0 +1,91 @@
+package com.pentalog.pentastagiu.homework2_bookCandyShop;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainClass {
+    public static void main(String[] args) {
+
+        Scanner scan=new Scanner(System.in);
+        String userEmail;
+        String password;
+        String userProduct="";
+        Boolean newProduct=true;
+        Integer userOption;
+        Double totalPrice=0.00;
+
+        //objects created
+        Customer bobleanca=new Customer(1001, "bobleanca94", "Bobleanca Dana", "bobleancadana@yahoo.com", MembershipStatus.NO_MEMBERSHIP );
+
+        Product book1=new Book("2001", "Effective Java\t\t\t\t", 50.00, "Joshua Bloch");
+        Product book2=new Book("2002", "Java: A Beginner’s Guide\t\t", 72.00, "Herbert Schildt");
+        Product book3=new Book("2003", "HTML and CSS:Design and Build Websites\t", 91.00, "John Ducket");
+        Product book4=new Book("2004", "Interactive Front-End Web Development\t", 32.00, "John Ducket");
+
+        Product candy1=new Candy("3001", "Candies with white chocolate Casali\t", 6.99, 100.00);
+        Product candy2=new Candy("3002", "Candies with taste and aroma of cherries Joy", 4.99, 100.00);
+        Product candy3=new Candy("3003", "Candies with chocolate and coffee Ferrero", 17.30, 225.00);
+        Basket basket=new Basket();
+
+        //create a list and adding products to list
+        List<Product> product=new ArrayList<>();
```

Comment:
cand denumim o colectie, e ok sa folosim denumirea la plural: products in cazul asta in loc de product